pybuildkite/pipelines.py

```diff
@@ -75,6 +75,28 @@ def create_pipeline(
 
         return self.client.post(self.path.format(organization), body=data)
 
+    def create_yaml_pipeline(
```

Comment:
To add a bit more detail, perhaps we formerly deprecate the previous call and log out that it will 500.

Reply:
One thing I forgot to mention is the old call (very likely) still works if the user hasn't migrated to YAML pipelines.

Reply:
One thing I forgot to mention is the old call (very likely) still works if the user hasn't migrated to YAML pipelines.

Reply:
Got it, that makes sense. In that case can we add specific documentation on this and the other method- perhaps referencing Buildkite's docs about the migration?

Reply:
Added the docs in https://github.com/pyasi/pybuildkite/pull/62/commits/445c4028d15e3ae33fed4fe642d3807364f4a052